editor/header/index.js

```diff
@@ -3,11 +3,26 @@
  */
 import './style.scss';
 import ModeSwitcher from './mode-switcher';
+import Dashicon from '../components/dashicon';
 
 function Header() {
 	return (
 		<header className="editor-header">
 			<ModeSwitcher />
+			<div className="editor-saved-state">
+				<Dashicon icon="yes" />
+				Saved
```

Comment:
There is some localization in the project now via `wp.i18n.__` but unsure if you wanted to hold off on these as temporary for now.